exec/java-exec/src/test/java/org/apache/drill/exec/util/FileSystemUtilTestBase.java

```diff
@@ -64,7 +63,9 @@
   @BeforeClass
   public static void setup() throws IOException {
     // initialize file system
-    fs = FileSystem.get(new Configuration());
+    Configuration configuration = new Configuration();
+    configuration.set(FileSystem.FS_DEFAULT_NAME_KEY, FileSystem.DEFAULT_FS);
+    fs = FileSystem.get(configuration);
```

Comment:
These three lines appear several times in the code, so may be it would be better to create a static method which returns `FileSystem` instance.

Reply:
Agree. Added static method `ExecTest.getLocalFileSystem`.